pshtt/pshtt.py

```diff
@@ -720,6 +722,11 @@ def is_hsts_preloaded(domain):
     return domain.domain in preload_list
 
 
+# Whether a domain's parent domain is in Chrome's HSTS preload list.
+def is_parent_hsts_preloaded(domain):
+    return parent_domain_for(domain.domain) in preload_list
```

Comment:
@h-m-f-t Just a note on my outstanding nit. ;) Should be a very quick refactoring change, if you concur.